libraries/helpers_nftables.rb

```diff
@@ -0,0 +1,122 @@
+module FirewallCookbook
+  module Helpers
+    module Nftables
+      include FirewallCookbook::Helpers
+
+      unless defined? CHAIN
+        CHAIN = {
+          in: 'INPUT',
+          out: 'OUTPUT',
+          pre: 'PREROUTING',
+          post: 'POSTROUTING',
+          # nil => "FORWARD",
+        }.freeze
+      end
+
+      unless defined? TARGET
```

Comment:
We can use `||=` here instead of the nested conditional

Reply:
done.